kentik_api_library/kentik_api/public/device_label.py

```diff
@@ -27,8 +27,8 @@ class DeviceLabel:
     _id: ID = DEFAULT_ID
     _user_id: Optional[ID] = DEFAULT_ID
     _company_id: ID = DEFAULT_ID
-    _created_date: str = DEFAULT_DATE
-    _updated_date: str = DEFAULT_DATE
+    _created_date: str = ""
+    _updated_date: str = ""
 
```

Comment:
The discussion involved adding getters for optional datetime: `def created(self) -> Optional[datetime]:` I think that they are much more usable than a string `def created_date(self) -> str:`. I would leave only getters returning datetime, as I don't see a reason for keeping 2 variants per field.
I think that this PR is a good place to do that (probably for all resources that have properties with dates)